_posts/2025-08-08-release-0_14_0.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Kroxylicious release 0.14.0"
+date: 2025-08-08 00:00:00 +1200
+author: "Rob Young"
+author_url: "https://github.com/robobario"
+# noinspection YAMLSchemaValidation
+categories: blog kroxylicious-proxy kroxylicious-operator releases
+tags: [ "releases", "kroxylicious-proxy", "kroxylicious-operator" ]
+---
+
+The Kroxylicious project is very pleased to announce the release
+of [Kroxylicious 0.14.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.14.0). See
+the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#0140) for a list of changes and
+summary of Deprecations, Changes and Removals.
+
+### Key Highlights
+
+* **New Authentication APIs for Plugin Authors:**
+  We are laying the groundwork for powerful client-identity-oriented Plugins. Our goal is to enable Plugins to access
+  and influence client identity. This opens the door to advanced features like per-user authorization, message
+  redaction, and much more!
+
+  This release takes the first major steps by adding APIs that allow Filters to:
+    - [Access client-provided TLS certificates](https://github.com/kroxylicious/design/blob/main/proposals/005-filter-api-to-expose-client-and-server-tls-info.md).
+    - [Report and access SASL authentication details](https://github.com/kroxylicious/design/blob/main/proposals/006-filter-api-to-expose-client-sasl-info.md)
+      across the Filter chain.
+
+* **New Back-Pressure Metrics:**
+  You can now pinpoint performance bottlenecks with new duration metrics that measure exactly how long the proxy is
+  applying back-pressure to client connections.
+
+* **Large Message Encryption Fix:**
+  See you later, alligator! We've squashed a critical bug in the Record Encryption Filter where encrypting events over
+  1MB would fail ([#2504](https://github.com/kroxylicious/kroxylicious/issues/2504)).
+
+* **API Versions Negotiation Fix:**
+  We resolved a bug that prevented some clients from connecting when Kroxylicious was proxying older Kafka broker
+  versions ([#2450](https://github.com/kroxylicious/kroxylicious/issues/2450)).
+
```

Comment:
Didn't we also make it work with Red Panda? If so, I think that's worth mentioning?